import-dashboards.py

```diff
@@ -223,6 +223,25 @@ def add_datasources(api_key):
             print ' * Cannot add ClickHouse Data Source'
             sys.exit(-1)
 
+    if 'PTSummary' not in ds:
+        print ' * PTSummary Data Source'
+        data = json.dumps({
+            'name': 'PTSummary',
+            'type': 'pmm-pt-summary-datasource',
+            'url': '',
+            'access': 'proxy',
+            'basicAuth': False,
+            'jsonData': {},
```

Comment:
Could you make sure that this variant is working? 
I can see that the datasource is added 

![Screenshot_20200826_125202](https://user-images.githubusercontent.com/25184101/91289419-fe8f3780-e79a-11ea-9b61-739b2744f277.png)


but other data source plugins are add with next fields
"jsonData":{"httpMethod":"POST","keepCookies":[],"timeInterval":"1s"}
Maybe it's required for Grafana?

Reply:
Yes it's working. It's not required here because it's not Grafana that is doing the request (since url is empty). It's handled by the data source plugin ([see here](https://github.com/percona/grafana-dashboards/pull/907/files#diff-024bfc19a9789f02c3cfed055c22e294R13)).

Reply:
I wonder if we should call it "PMM pt-summary" or something like that in UI